.github/workflows/nightly-build.yaml

```diff
@@ -0,0 +1,40 @@
+name: Nightly Build
+on:
+  schedule:
+    - cron: '0 20 * * *'
+  workflow_dispatch:
+    inputs:
+      debug_enabled:
+        type: boolean
+        description: 'Run the build with tmate debugging enabled (https://github.com/marketplace/actions/debugging-with-tmate)'
+        required: false
+        default: false
+jobs:
+  run-flink-derived-feature-view-example:
```

Comment:
Can we have a script that programmatically enumerates folders that match certain pattern and execute the scripts defined in these folders, instead of  having to add an entry in this file for each test?